extensions/amp-video-iframe/0.1/test/test-amp-video-iframe.js

```diff
@@ -0,0 +1,322 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as sinon from 'sinon';
+
+import '../amp-video-iframe';
+import {Services} from '../../../../src/services';
+import {VideoEvents} from '../../../../src/video-interface';
+import {
+  addAttributesToElement,
+  whenUpgradedToCustomElement,
+} from '../../../../src/dom';
+import {htmlFor} from '../../../../src/static-template';
+import {listenOncePromise} from '../../../../src/event-helper';
+import {tryParseJson} from '../../../../src/json';
+
+function getIntersectionMessage(id) {
+  return {data: {id, method: 'getIntersection'}};
+}
+
+describes.realWin('amp-video-iframe', {
+  amp: {
+    extensions: ['amp-video-iframe'],
+  },
+}, env => {
+
+  let win;
+  let doc;
+  let videoManagerStub;
+
+  beforeEach(() => {
+    win = env.win;
+    doc = win.document;
+
+    videoManagerStub = {
+      register: env.sandbox.spy(),
+    };
+
+    env.sandbox.stub(Services, 'videoManagerForDoc').returns(videoManagerStub);
+  });
+
+  function getIframeSrc() {
+    const {port} = location;
+    return (
+      `http://iframe.localhost:${port}/test/fixtures/served/video-iframe.html`);
+  }
+
+  function createVideoIframe(size = null) {
```

Comment:
Does the `null` default value add anything? `size` is undefined by default, which is falsy, so the default value `null` looks redundant when there is only a falsy check.

Reply:
#16550 :)

Reply:
Thanks! There are a few other places in this PR with the same pattern, but it's not super important. To me, it adds a bit of overhead when a reader has to evaluate whether the `= null` default value is significant or not.

Reply:
Could you point to some other examples where the default value is extraneous, or the optional parameter pattern is followed?

Reply:
Sure. It looks like `undefined` and `null` would give the same behavior in these situations, which makes the `= null` unnecessary:

https://github.com/ampproject/amphtml/blob/6ec9b0555074f006558d362961842e0060b94a1a/src/iframe-video.js#L77-L88

https://github.com/ampproject/amphtml/blob/6ec9b0555074f006558d362961842e0060b94a1a/src/video-iframe-integration.js#L309-L322

Reply:
Thanks! #16563